source/code/include/playfab/PlayFabEventRouter.h

```diff
@@ -37,8 +38,10 @@ namespace PlayFab
     class PlayFabEventRouter : public IPlayFabEventRouter
     {
     public:
-        PlayFabEventRouter();
+        PlayFabEventRouter(bool threadedEventPipeline);
         virtual void RouteEvent(std::shared_ptr<const IPlayFabEmitEventRequest> request) const;
+
+        virtual void Update() override;
```

Comment:
Needs comments indicating that this function should only be called when threadedEventPipeline is false.